pytorch_lightning/callbacks/progress/tqdm_progress.py

```diff
@@ -381,7 +381,9 @@ def convert_inf(x: Optional[Union[int, float]]) -> Optional[Union[int, float]]:
     return x
 
 
-def _update_n(bar: _tqdm, value: int) -> None:
-    if not bar.disable:
```

Comment:
why not check disable anymore?

Reply:
because all of them are under `_should_update` check which already checks this.

Reply:
good catch!